addon/components/pix-pagination.js

```diff
@@ -104,18 +104,28 @@ export default class PixPagination extends Component {
     return Math.min(rowCount, this.firstItemPosition + this.pageSize - 1);
   }
 
+  @action
+  onChange() {
+    if (typeof this.args.onChange !== 'function') return;
+
+    this.args.onChange();
```

Comment:
**question:** N'est-il pas préférable de laisser le navigateur péter une erreur tout seul si le `onChange` de la ligne `this.args.onChange()` n'est pas une fonction ?

Dans ce cas, je trouve que l'ajout de cette action n'est pas forcément nécessaire, on pourrait utiliser directement la prop.

Reply:
**proposition:** aussi, ce serait pas mal que l'action renvoie les infos de pagination (ne serait-ce que la nouvelle page en cours) ?
C'est plus efficace que de devoir retrouver à la main les infos dans les query params

Reply:
Comme le onChange n'est pas obligatoire.je n'ai pas envie que le navigateur pète si c'est autre chose qu'une function (j'aurai pu ajouter un ember/warn au constructor)

Pour le moment c'etait plus le fait d'être ISO avec un besoin qu'on a sur PixOrga. (Permettre d'exécuter une fonction au changement de la page, reset de tableau de suppression de ligne)

Le changement est intéressant. Mais qui nécessitera un breaking change, et un alignement avec les équipes si l'on ne doit plus passer par le router dans le PixPagination mais seulement par le onChange.